pkg/cache/cache_test.go

```diff
@@ -143,46 +143,47 @@ func TestCacheWriteCacheDirExists(t *testing.T) {
 }
 
 func TestStoreRelocationMapping(t *testing.T) {
-	tests := []struct {
+	tests := map[string]struct {
 		name              string
 		relocationMapping relocation.ImageRelocationMap
 		tag               cnab.OCIReference
 		bundle            cnab.ExtendedBundle
 		wantedReloPath    string
 		err               error
 	}{
-		{
-			name:   "relocation file gets a path",
+		"relocation file gets a path": {
 			bundle: cnab.ExtendedBundle{},
 			tag:    kahn1dot01,
 			relocationMapping: relocation.ImageRelocationMap{
 				"asd": "asdf",
 			},
 			wantedReloPath: path.Join("/.porter/cache", kahn1dot0Hash, "cnab", "relocation-mapping.json"),
 		},
-		{
-			name:           "no relocation file gets no path",
+		"no relocation file gets no path": {
 			tag:            kahnlatest,
 			bundle:         cnab.ExtendedBundle{},
 			wantedReloPath: "",
 		},
 	}
 
-	for _, tc := range tests {
-		t.Run(tc.name, func(t *testing.T) {
+	for name, test := range tests {
+		test := test
+		t.Run(name, func(t *testing.T) {
 			t.Parallel()
-			tc := tc
 
 			cfg := config.NewTestConfig(t)
 			c := New(cfg.Config)
+			cb, err := c.StoreBundle(cnab.BundleReference{Reference: test.tag, Definition: test.bundle, RelocationMap: test.relocationMapping})
+			assert.NoError(t, err, fmt.Sprintf("didn't expect storage error for test %s", test.name))
+			s := cb.RelocationFilePath
+			if s != "" {
+				s = cb.RelocationFilePath[12:len(cb.RelocationFilePath)]
```

Comment:
I have to do this for the following reasons. 

When running this test, you'll get the `/home/myuser/.` HOME directory and not the directory on disk of the user running the test.  

This is from the setup text fixture that does this:
` home := "/home/myuser/.porter"` 

I can set the PORTER_HOME to be `$HOME/.porter` to make this better.

Reply:
Done.